docs/integrations/enterprise-connectors/source-workday.md

```diff
@@ -18,6 +19,70 @@ Airbyte's incubating Workday enterprise source connector currently offers the fo
 | SSL connection                | Yes        |
 | Namespaces                    | No         | 
 
+## Prerequisites
```

Comment:
Docusaurus is weird about liking blank lines. I'll spare you the painful details but I'd say if we get into the habit of placing an empty line between different markup elements, things will go easier for us down the road when we have more automation.

Reply:
Noted, added!